lambda_code/src/fhir_controller.py

```diff
@@ -1,36 +1,67 @@
 import json
 import re
 import uuid
+from typing import Optional
 
+from fhir.resources.operationoutcome import OperationOutcome
+
+from fhir_repository import ImmunisationRepository, create_table
 from fhir_service import FhirService
-from models.errors import Severity, Code, create_operation_outcome
+from models.errors import Severity, Code, create_operation_outcome, ResourceNotFoundError, UnhandledResponseError
+
+
+def make_controller():
+    imms_repo = ImmunisationRepository(create_table())
+    service = FhirService(imms_repo=imms_repo)
+    return FhirController(fhir_service=service)
 
 
 class FhirController:
-    immunisation_id_pattern = r"^[A-Za-z0-9\-.]{1,64}$"
+    immunization_id_pattern = r"^[A-Za-z0-9\-.]{1,64}$"
```

Comment:
I think you have an American English dictionary installed. This and every other usage of the word in here should be "immunisation" with an "s".